restClient/stp.rest

```diff
@@ -0,0 +1,517 @@
+
+
+### testing upload file example
+POST http://localhost:5000/SASjsApi/stp/execute/?_program=/Public/app/viya/services/editors/loadfile&table=DCCONFIG.MPE_X_TEST
```

Comment:
wouldn't this service need to be deployed prior to testing?

If so we should probably deploy a dummy service to a /tests/ location or similar

Reply:
I have just put a sample example, if anyone wants to manually test their api server, they can use this instead of preparing `postman`.